src/event/EventLoop.h

```diff
@@ -65,19 +65,23 @@ class EventLoop
     };
     using ConnectKey = uint32_t;
 
-    virtual void* handle() const;
+    void* handle() const;
 
 private:
-    void run_internal();
-    void post_internal(std::unique_ptr<Event>&& event);
-    void stop_internal();
-    uint32_t startTimer_internal(const std::shared_ptr<Event>& event, uint64_t timeout, TimerMode mode);
-    void stopTimer_internal(uint32_t id);
-    bool processEvents();
-
-private:
-    std::mutex mMutex;
-
+    struct EventLoopImpl
```

Comment:
This should become EventLoop and the current EventLoop should be EventLoopUV and the GLFW one should become EventLoopGLFW